Libs/MRML/Core/vtkMRMLClipModelsNode.cxx

```diff
@@ -128,5 +144,41 @@ void vtkMRMLClipModelsNode::PrintSelf(ostream& os, vtkIndent indent)
   os << indent << "YellowSliceClipState: " << this->YellowSliceClipState << "\n";
   os << indent << "GreenSliceClipState:  " << this->GreenSliceClipState << "\n";
   os << indent << "RedSliceClipState:    " << this->RedSliceClipState << "\n";
+  os << indent << " clippingMethod=\"" << (this->GetClippingMethodAsString(this->ClippingMethod)) << "\n";
+}
+
+//-----------------------------------------------------------------------------
+vtkMRMLClipModelsNode::ClippingMethodType vtkMRMLClipModelsNode::GetClippingMethodFromString(const char* name)
+{
+  if (name == NULL)
+    {
+    // invalid name
+    return ClippingMethod_Last;
```

Comment:
Since we are typedef'ing the enum as ClippingMethodType, we can't return an int (I tried at first and it would not compile).